LICENSE

```diff
@@ -1,7 +1,7 @@
 -------------------------------------------------------------------- 
-                 The Xdebug License, version 1.02
-             (Based on "The PHP License", version 3.0)
-   Copyright (c) 2003-2018 Derick Rethans. All rights reserved.
+                 The Xdebug License, version 1.03
+             (Based on "The PHP License", version 3.01)
+   Copyright (c) 2003-2022 Derick Rethans. All rights reserved.
```

Comment:
Perhaps simpler to drop (as in PHP) the date range

Reply:
The date range here is for the copyright of the *license*, and not the code. I don't care either way though. I do sort of like having the earliest year in it :-).